docs/business-finder-tagging.md

```diff
@@ -0,0 +1,24 @@
+# EU Exit Readiness Business Finder Tagging
+
+This document explains why the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) ("business finder") uses [content-tagger](https://github.com/alphagov/content-tagger) 'links' to filter the finder.
+
+TODO - explain links vs tags properly, from an editor workflow perspective.
+
+Usually, finders use facets that have been published as part of the content item itself. These appear under `details["facets"]` in the [content item JSON](https://www.gov.uk/api/content/cma-cases). Search results get more granular as you select values in different facets (the number of results goes down).
+
+The business finder uses facets that reference 'links' published by content-tagger, and that are [manually grouped in a YAML file](https://github.com/alphagov/content-tagger/blob/master/lib/data/find-eu-exit-guidance-business.yml). Search results get more _broad_ as you select values in different facets (number of results goes up). Facets appear under `links["facet_group"]` in the [business finder content item JSON](https://www.gov.uk/api/content/find-eu-exit-guidance-business).
+
+Let's call these "content-based facets" and "link-based facets" respectively.
+
+Content-based facets are part of the content item itself, so when the content item is updated via its publishing API, they're updated too and associated with the change note accompanying the change. This triggers an email to subscription lists which are subscribed to that facet, and the change note appears in the email body.
+
+This works well for normal finders, as they only contain content published by one publishing app, i.e. content is updated and tagged in the same publisher.
+
+The business finder, however, can contain content from any publishing app. This poses two problems:
+
+1. If we were to add the facets in the usual way, we would have had to add an interface to every publishing app to do it.
+2. If we add or remove facets in a content item, this will trigger an email update when we may not want it to. (Would it? - or could we disable the email checkbox?)
```

Comment:
@leenagupte could you answer this?

Reply:
This isn't really a problem now as we've added the checkbox, but it was a consideration when we were considering approaches.